backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/Subscription.java

```diff
@@ -2,19 +2,27 @@
 
 import org.immutables.value.Value.Immutable;
 
+import io.micrometer.common.lang.Nullable;
```

Comment:
Use the standard `jakarta.annotation.Nullable` here instead of the micrometer one. Both work, but using the jakarta one is more future-proof.

We will probably remove micrometer from the project in favor of OpenTelemetry in a future PR.

Reply:
fixed